src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,107 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        table = (Node<K, V>[])new Node[INITIAL_CAPACITY];
+        threshold = (int)(INITIAL_CAPACITY * LOAD_FACTOR);
+    }
+
+    private static class Node<K, V> {
+        private final K key;
+        private V value;
+        private Node<K,V> next;
+
+        Node(K key, V value, Node<K, V> next) {
```

Comment:
```suggestion
        Node(K key, V value) {
```
в тебе все одно завжди при ініціалізації `next = null`